### DISCUSSION TITLE
[Feat]: Web Accessibility (WAI)

### DISCUSSION THREAD

**Discussion by bozdevs** [davidjerleke/embla-carousel#927] [Ideas]
### Which variants of Embla Carousel are relevant to this feature request?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### Feature description

Accessibility markings should be added. (Plugin or Core)

### Additional Context

[https://www.w3.org/WAI/roles/developers/](https://www.w3.org/WAI/roles/developers/)

### Before submitting

- [x] I've made research efforts and searched the documentation
- [X] I've searched for existing issues
- [X] I agree to follow this project's Contributing Guidelines for feature requests

**Comment by meirroth** [comment]
@bozdevs Embla Carousel is designed to be fully customizable and leaves UI details, including accessibility, up to the developer.

**Comment by davidjerleke** [comment]
@bozdevs please search for existing discussions/issues first before creating a lot of new ones. 

- Duplicate of: #6.
